addons/website/static/src/less/website.backendDashboard.less

```diff
@@ -137,6 +137,7 @@
                 height: 80px;
                 overflow: hidden;
                 text-overflow: ellipsis;
+                white-space: nowrap;
```

Comment:
It fixes that :
https://drive.google.com/file/d/0B0IF0OmAMb5OdXM0cnhGTnJMOXc/view